extensions/wikia/VideoHandlers/filerepo/WikiaLocalFileShared.class.php

```diff
@@ -289,9 +289,19 @@ function metaValue( $key, $default = '' ) {
 			}
 		}
 
-		if ( !empty($this->metadata[$key]) ) {
-			$value = $this->metadata[$key];
+		if ( is_array( $this->metadata ) ) {
+			if ( !empty( $this->metadata[$key] ) ) {
+				$value = $this->metadata[$key];
+			} else {
+				$value = $default;
+			}
 		} else {
+			\Wikia\Logger\WikiaLogger::instance()->error(
+				'File metadata not an instance of an array. Expecting array', [
+					'type' => gettype( $this->metadata ),
+					'file' => $this->oFile
+				]
```

Comment:
You can add `'exception' => new Exception()` to log the backtrace for easier debugging.